### PR TITLE
fix path and refactor a bit

### PR DESCRIPTION
**Pull request by ninyawee**

### CODE REVIEW

File: recport/portfolio.py [hunk 1]
```diff
@@ -15,26 +15,35 @@ def __init__(self, port_name):
...
-                file_directory.mkdir(parents=True)
-            with Path(file_path).open(mode='w') as f:
-                
-                f.write(doc.as_string())
+        raise NotImplementedError
+
+    def updateTofile(self):
+        """ Update/Save data to file, create one if not exists """
+        port_name = self.port_name
+        file_path = self.file_path
```
Comment (harit559) [inline recport/portfolio.py:38]: This standard helps reduce access overhead

File: recport/portfolio.py [hunk 2]
```diff
@@ -15,26 +15,35 @@ def __init__(self, port_name):
...
-            doc = document()
-            doc.add('port_name',port_name)
-            if not file_directory.exists():
-                file_directory.mkdir(parents=True)
-            with Path(file_path).open(mode='w') as f:
-                
-                f.write(doc.as_string())
+        raise NotImplementedError
+
+    def updateTofile(self):
```
Comment (ninyawee) [inline recport/portfolio.py]: This is should be `updateToFile()`

File: recport/portfolio.py [hunk 3]
```diff
@@ -51,8 +60,6 @@ def deposit(self):
     def withdraw():
         pass
 
-if __name__ == "__main__":
-    portfolio = Portfolio('jack')
-
-
 
+if __name__ == "__main__":
+    portfolio = Portfolio("jack")
```
Comment (ninyawee) [inline recport/portfolio.py:65]: wow, nice format

File: recport/portfolio.py [hunk 4]
```diff
@@ -51,8 +60,6 @@ def deposit(self):
     def withdraw():
```
Comment (harit559) [inline recport/portfolio.py:60]: This needs arguments

**Review by ninyawee — COMMENTED** [review]
This should be in tests

**Review by harit559 — CHANGES_REQUESTED** [review]
something

**Pull request merged**